D1673/blas_interface.md

```diff
@@ -2178,13 +2207,19 @@ real element types), `xDOTC`, and `xDOTU` (for complex element types).
 * *Requires:*
 
   * `T` shall be *Cpp17MoveConstructible*.
+
   * `init + v1(0)*v2(0)` shall be convertible to `T`.
-  * `v1` and `v2` have the same domain.
```

Comment:
Did you mean to replace all occurrences of "have the same domain"?
I am asking because I noticed there is at least one in `givens_rotation_apply()`

Reply:
@dalg24 I'm still thinking about that ;-) .  The reason I changed the other occurrences was to have the Requires and Mandates clauses match, so that it would be easier to read the Mandates clauses.  I could also replace the other instances of "have the same domain."

Reply:
@dalg24 Christian thinks we should change the "have the same domain" things first.  I'll do that.

Reply:
Done!